include/deal.II/numerics/time_dependent.h

```diff
@@ -524,7 +525,7 @@ class TimeDependent
    * the TimeStepBase class.
    *
    * Instead of using the above form, you can equally well use
-   * <tt>bind2nd(mem_fun1(&X::unary_function), arg)</tt> which lets the @p
+   * <tt>std_cxx11::bind(&X::unary_function, objX, arg)</tt> which lets the @p
```

Comment:
I am that familiar with bind, but since `&X::unary_function` refers to a member function, don't we need to use `mem_fn` (that is, the C++11 replacement for `mem_fun`, note the `u`) here?